include/fastdds/utils/IPLocator.hpp

```diff
@@ -102,6 +102,11 @@ class IPLocator
             const Locator_t& locator,
             unsigned char* dest);
 
+    //! Copies locator's IPv4.
```

Comment:
It would be nice if we could doxygen both `copyIPv4` methods. State that the first one is meant to be use to retrieve the address to a buffer while the second one among locators.